client/src/pages/Login/Login.tsx

```diff
@@ -37,24 +38,25 @@ export default function Login(): JSX.Element {
   };
 
   return (
-    <Grid container component="main" className={classes.root}>
-      <CssBaseline />
-      <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
-        <Box className={classes.authWrapper}>
-          <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />
-          <Box width="100%" maxWidth={450} p={3} alignSelf="center">
-            <Grid container>
-              <Grid item xs>
-                <Typography className={classes.welcome} component="h1" variant="h5">
-                  Welcome back!
-                </Typography>
+    <Dialog open fullWidth maxWidth="sm">
+      <Grid container component="main" className={classes.root}>
+        <CssBaseline />
+        <Grid item xs={12} sm={12} md={12} lg={12} elevation={6} component={Paper} square>
+          <Box className={classes.authWrapper}>
+            <Box width="67%" maxWidth={600} alignSelf="center">
```

Comment:
Try to avoid having inline styling, even for properties like width

Reply:
thank you for the comment. Changed the element's inline styling to useStyle sheet.